### DISCUSSION TITLE
Mochi broker logo

### DISCUSSION THREAD

**Discussion by leandrofars** [mochi-mqtt/server#241] [General]
Is there any symbol, or logo of mochi broker? I would like to represent mochi broker in a visual way on the sketch of my project topology

**Comment by mochi-co** [comment]
Hi @leandrofars - sorry for the delay. There's currently no visual logo for mochi broker, but feel free to stylise it as you like for your project :)

**Comment by mochi-co** [comment]
@leandrofars If it's any use for you still, there's some logos now available in https://github.com/mochi-mqtt/assets 👍🏻

**Comment by leandrofars** [comment]
@mochi-co Amazing! going to update my project schema: https://github.com/leandrofars/oktopus with your logo, thanks!
